README.md

```diff
@@ -16,30 +16,34 @@ optionally air-gapped datacenter or in the cloud.
 
 ### Kubernetes
 
-[Kubernetes (k8s)](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/) is an open-source system for automating deployment, scaling, and management of containerized applications.
+Kubernetes (K8s) is an open-source system for automating deployment, scaling, and management of containerized applications.
+
+Consult our [Kubernetes Guide](docs/kubernetes-cluster.md) to build a GPU-enabled Kubernetes cluster.
 
-See the [Kubernetes Guide](docs/kubernetes-cluster.md) for information on building a GPU-enabled Kubernetes cluster
+For more information on Kubernetes in general, refer to the [official Kubernetes docs](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/).
 
 ### Slurm
 
-[Slurm](https://slurm.schedmd.com/overview.html) is an open source, fault-tolerant, and highly scalable cluster management and job scheduling system for large and small Linux clusters.
+Slurm is an open source, fault-tolerant, and highly scalable cluster management and job scheduling system for large and small Linux clusters.
+
+Consult our [Slurm Guide](docs/slurm-cluster.md) to build a GPU-enabled Slurm cluster.
 
-See the [Slurm Guide](docs/slurm-cluster.md) for information on building a GPU-enabled Slurm cluster
+For more information on Slurm in general, refer to the [official Slurm docs](https://slurm.schedmd.com/overview.html).
 
 ### DGX POD Hybrid Cluster
 
-Hybrid Kubernetes and Slurm DGX cluster based on the DGX POD reference architecture
+A hybrid cluster with both Kubernetes and Slurm can also be deployed. This is recommended for DGX Pod and other setups that wish to make maximal use of the cluster.
 
-See the [Deployment Guide](docs/dgx-pod.md) for step-by-step instructions and setup information
+Consult our [DGX Pod Guide](docs/dgx-pod.md) for step-by-step instructionson setting up a hybrid cluster.
 
 For more information on deploying DGX in the datacenter, consult the
 [DGX Data Center Reference Design Whitepaper](https://nvidia-gpugenius.highspot.com/viewer/5b33fecf1279587c07d8ac86)
 
 ### Virtual
```

Comment:
What do you think about reordering thing in the readme to be:
- Virtual
- DGX POD
- Kupernetes
- Slurm

I envision people going next to the virtual or DGX POD docs next. Thoughts?

Reply:
Instead of "Deployment Options" would "Deployment Examples" be more accurate?

Reply:
Sorry I realize my comment was based on existing Readme.md not the PR version.